spyder_remote_services/app.py

```diff
@@ -0,0 +1,118 @@
+"""Jupyter server example application."""
+
+import os
```

Comment:
```suggestion
from contextlib import suppress
import os
```

Move standard library import that's placed below to this section.